*** .oh-my-zsh/themes/agnoster.zsh-theme	2014-07-27 17:00:22.137778000 +0900
--- agnoster.zsh-theme	2014-07-27 17:04:07.984434873 +0900
***************
*** 26,32 ****
  # A few utility functions to make it easy and re-usable to draw segmented prompts
  
  CURRENT_BG='NONE'
! SEGMENT_SEPARATOR='⮀'
  
  # Begin a segment
  # Takes two arguments, background and foreground. Both can be omitted,
--- 26,32 ----
  # A few utility functions to make it easy and re-usable to draw segmented prompts
  
  CURRENT_BG='NONE'
! SEGMENT_SEPARATOR=''
  
  # Begin a segment
  # Takes two arguments, background and foreground. Both can be omitted,
***************
*** 100,106 ****
      zstyle ':vcs_info:*' formats ' %u%c'
      zstyle ':vcs_info:*' actionformats ' %u%c'
      vcs_info
!     echo -n "${ref/refs\/heads\//⭠}${vcs_info_msg_0_%% }${mode}"
    fi
  }
  
--- 100,106 ----
      zstyle ':vcs_info:*' formats ' %u%c'
      zstyle ':vcs_info:*' actionformats ' %u%c'
      vcs_info
!     echo -n "${ref/refs\/heads\// }${vcs_info_msg_0_%% }${mode}"
    fi
  }
  
***************
*** 179,181 ****
--- 179,182 ----
  }
  
  PROMPT='%{%f%b%k%}$(build_prompt) '
+ 
