*** fish_prompt.fish	2015-06-10 18:17:58.000000000 +0900
--- fish_prompt.fish	2015-06-10 18:18:56.000000000 +0900
***************
*** 14,21 ****
  
  
  set -g current_bg NONE
! set segment_separator \uE0B0
! set right_segment_separator \uE0B0
  # ===========================
  # Helper methods
  # ===========================
--- 14,21 ----
  
  
  set -g current_bg NONE
! set segment_separator \u2b80
! set right_segment_separator \u2b80
  # ===========================
  # Helper methods
  # ===========================
***************
*** 114,120 ****
        set -l branch (command git show-ref --head -s --abbrev |head -n1 2> /dev/null)
        set ref "➦ $branch "
      end
!     set branch_symbol \uE0A0
      set -l branch (echo $ref | sed  "s-refs/heads/-$branch_symbol -")
      if [ "$dirty" != "" ]
        prompt_segment yellow black "$branch $dirty"
--- 114,120 ----
        set -l branch (command git show-ref --head -s --abbrev |head -n1 2> /dev/null)
        set ref "➦ $branch "
      end
!     set branch_symbol \u2b60
      set -l branch (echo $ref | sed  "s-refs/heads/-$branch_symbol -")
      if [ "$dirty" != "" ]
        prompt_segment yellow black "$branch $dirty"
